profiling_router.go

```diff
@@ -0,0 +1,147 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"strconv"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/go-utils/logger"
+)
+
+const PprofRoutePrefix = "/debug/pprof"
```

Comment:
question: does it really need to be public? 🤔 
```suggestion
const pprofRoutePrefix = "/debug/pprof"
```